packages/toolkit/src/codewhisperer/models/constants.ts

```diff
@@ -316,32 +317,46 @@ export const viewProposedChangesMessage =
 
 export const dependencyDisclaimer =
     'Please confirm you are ready to proceed with the transformation. Amazon Q Code Transformation will upload the application code and its dependency binaries from your machine to start the upgrade. If you have not yet compiled the application on your local machine, please do so once before starting the upgrade. Install Maven to ensure all module dependencies are picked for Transformation.'
 
 export const dependencyFolderName = 'transformation_dependencies_temp_'
 
 export const installErrorMessage =
-    'Failed to execute Maven install. It is possible that the upload does not include all dependencies. We will still attempt to complete the transformation.'
+    'Failed to execute Maven install. Please see: https://docs.aws.amazon.com/amazonq/latest/aws-builder-use-ug/code-transformation.html'
 
 export const dependencyErrorMessage =
-    'Failed to execute Maven copy-dependencies. It is possible that the upload does not include all dependencies. We will still attempt to complete the transformation.'
+    'Failed to execute Maven copy-dependencies. Please see: https://docs.aws.amazon.com/amazonq/latest/aws-builder-use-ug/code-transformation.html'
 
 export const planIntroductionMessage =
     'We reviewed your Java JAVA_VERSION_HERE application and generated a transformation plan. Any code changes made to your application will be done in the sandbox so as to not interfere with your working repository. Once the transformation job is done, we will share the new code which you can review before acccepting the code changes. In the meantime, you can work on your codebase and invoke Q Chat to answer questions about your codebase.'
 
 export const planDisclaimerMessage = '**Proposed transformation changes** \n\n\n'
 
-export const enterJavaHomeMessage = 'Enter the value of JAVA_HOME for Java'
+export const enterJavaHomeMessage = 'Enter the JAVA_HOME of JDK'
+
+export const windowsJavaHomeHelpMessage =
+    'In a terminal, try running "cd C:\\Program Files\\Java" and then "dir" to see a list of installed JDKs. Then "cd" into JDK JAVA_VERSION_HERE (if present) and run "cd" once more to get the JAVA_HOME. It should look something like "C:\\Program Files\\Java\\jdk-11.0.1"'
+
+export const nonWindowsJava8HomeHelpMessage =
+    'Try running "/usr/libexec/java_home -v 1.8" in a terminal to find this. It should look something like "/Library/Java/JavaVirtualMachines/jdk-8.jdk/Contents/Home"'
+
+export const nonWindowsJava11HomeHelpMessage =
+    'Try running "/usr/libexec/java_home -v 11" in a terminal to find this. It should look something like "/Library/Java/JavaVirtualMachines/jdk-11.jdk/Contents/Home"'
+
+export const projectSizeTooLargeMessage =
```

Comment:
Are these approved text? It seems like this error message is vague. What limit? Whos limit?

Eg) `Your project size exceeds the Amazon Q Code Transform upload limit of 1GB`

Reply:
Will change the text to match that error message you suggested. I think we can fine-tune the text later but there's other important work that shouldn't be blocked by perfecting the strings.

And yes it is used in `transformByQHandler.ts` -- you have to click 'View Diff' for that since it's not shown by default as it's too large.